src/pages/home.js

```diff
@@ -1,12 +1,55 @@
+// home.js
+
 import React from "react";
-import Stack from "react-bootstrap/Stack";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
 import Card from "../components/card";
+import { FaChartLine } from "react-icons/fa6";
+import { SlLogin } from "react-icons/sl";
+import { FcFile } from "react-icons/fc";
 
 function Home() {
   return (
-    <Stack gap={3}>
-      <Card title="Daily Logins" content={150} label=" Since Last Month" />
-    </Stack>
+    <Row>
```

Comment:
There is an opportunity to improve here. If we continue to do this, our "Home" page will do many things

* Its managing and rendering "Overview"
* It will also manage the "Sessions Analytics" and the "Rosters" table.

Can you think of how we can improve this so that it only renders them instead of managing it as well?